sphinx-docs/developer_docs/index.rst

```diff
@@ -1,3 +1,34 @@
+Setting up your development environment
+=======================================
+
+KA Lite is like a normal django project, if you have done Django before, you will recognize most of these steps.
+
+  1. Check out the project from our _`github`
+  1. Create a virtual environment "kalite" that you will work in:
+     .. ::
+       sudo pip install virtualenvwrapper
+       mkvirtualenv kalite
+       workon kalite
+  1. Install kalite in your virtualenv in "editable" mode, meaning that the source is just linked:
+     .. ::
+       cd path/to/repo
+       pip install -e .
+  1. Install additional development tools:
+     .. ::
+       pip install -r requirements.txt
+  1. Run a development server and use development settings like this:
+     .. ::
+       kalite manage runserver --settings=kalite.project.settings.dev
+  
+
+You can also change your ``~/.kalite/settings.py`` to point to ``kalite.project.settings.dev`` by default, then you do not have to specify `--settings=...` every time you run kalite.
+
+Now, everytime you work on your development environment, just remember to switch on your virtual environment with `workon kalite`. This can be done automatically with _`oh-my-zsh`.
```

Comment:
Hmm I remember there's additional set up required for `oh-my-zsh` to automatically switch virtualenvs. I prefer that we either discuss that additional step further (adding `pip` to zsh modules), or we just omit any mention of `of-my-zsh` (since that requires telling them to switch shells as well).